src/ResourceManager/Compute/Commands.Compute/Extension/Chef/SetAzureVMChefExtension.cs

```diff
@@ -101,6 +103,18 @@ public string TypeHandlerVersion
         public string BootstrapOptions { get; set; }
 
         [Parameter(
+           ValueFromPipelineByPropertyName = true,
```

Comment:
I have a dependency on #2173 for adding automated test in Resource management i.e. for Set-AzureRMVMChefExtension so I have created separate issue #3147 for this.